### ISSUE TITLE
In ProxylessNAS Search Mode , Accuarcy and loss does not change after several epochs 

### ISSUE THREAD

**Issue by Henrylol** [microsoft/nni#4860]
**Describe the issue**:
As title described, I ran  ProxylessNAS  example with slight network parameter modification.  After 5 epochs, acc 1,  acc 5  looks like the same as what I got in previous epochs, Two screenshots show below described this situation. If acc won't increase, how can I know when to get a good archtecture produced by Proxyless Network ？

![Epoch_1](https://user-images.githubusercontent.com/12585877/168234194-32e105e0-979d-4d68-8ee8-2702cc63da2b.jpg)

![Epoch_2](https://user-images.githubusercontent.com/12585877/168234195-4061fba7-d0ff-42a5-9616-cf291bcd33de.jpg)

**Environment**:
- NNI version: 2.7
- Training service (local|remote|pai|aml|etc): local
- Client OS:  Ubuntu 18.04
- Server OS (for remote mode only):
- Python version:  3.9
- PyTorch/TensorFlow version:  Pytorch 1.10
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?: No

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by mahdihey** [comment]
I have a similar issue running ProxylessNAS example.

**Comment by goelbenj** [comment]
Has anyone been able to solve this? I'm experienced similar behaviour with the default search hyper parameters.
